sockeye/arguments.py

```diff
@@ -249,7 +249,15 @@ def add_model_parameters(params):
 
     model_params.add_argument('--weight-tying',
```

Comment:
what's wrong with tying? After all that's also what's it called in "Using the Output Embedding to Improve Language Models". Is this really a reason to break backwards compatibility?